src/otel/otel.ts

```diff
@@ -38,23 +37,17 @@ export class ElementCallOpenTelemetry {
   private _provider: WebTracerProvider;
   private _tracer: Tracer;
   private _anonymity: Anonymity;
+  private _otlpExporter: OTLPTraceExporter;
 
   static globalInit(): void {
-    settingsBus.on("opt-in-analytics", recheckOTelEnabledStatus);
-    recheckOTelEnabledStatus(getSetting("opt-in-analytics", false));
+    recheckOTelEnabledStatus();
```

Comment:
Since this is now always enabled, and the config can't change, could we get rid of the whole recheck-loop thing and move the initialization to inside `get instance`?

Reply:
Yeah, may as well I guess - I assume we'll want a config setting at some point, but it can come back later. I've kept the globalInit since I think it's nice that it only reads from the config once the config is ready. wdyt?

Reply:
Oh, sure, that's fine